### ISSUE TITLE
i updated and crashed my web

### ISSUE THREAD

**Issue by ocarito** [tlovett1/custom-contact-forms#104]
I am setting up a new website, working good looking good.. the form was nice and neat... but then today I saw that the Contact Form Plugin needed an update...but without backing up anything I just did it.. and BAM my web is down. 

I got this:

_______
Internal Server Error

The server encountered an internal error or misconfiguration and was unable to complete your request.

Please contact the server administrator at webmaster@mywebsite.com to inform them of the time this error occurred, and the actions you performed just before this error.

More information about this error may be available in the server error log.

Additionally, a 404 Not Found error was encountered while trying to use an ErrorDocument to handle the request.
_________________

Can't do anything now... any idea of where to look? or should I reinstall and redo my whole work? I was looking for the sql lines that I may reverse on my db, but there is nothing so far at hand to use. 

any Help?

**Comment by tlovett1** [comment]
You should just restore the old version of the plugin. Can you tell me what's in the error logs?

**Comment by ocarito** [comment]
so there is no modification to the Database when updating the plugin? I installed the plugin from straight from the repository... so I do not have a old copy of the plugin.. what I did first was to rename the folder, and nothing, then I went to the database to the ws_option plugins and erase it's content, nothing either. kind of I need to re install all again. - in which folder should I look for the log file?

**Comment by ocarito** [comment]
I found a backup I did couple of weeks ago, and i restored the old folder plugin.. and nothing happened.. still having the same issue. i will do now the db :(

**Comment by ocarito** [comment]
I restored the plugin and nothing happened, the problem was the same. I found an old month backup and I restored altogether with the db and now it's working.. bad I didn't backup my info before the update. - there should be a little sql script on your repository to fix this things. - in another website I installed your script I had a conflct because I am using .htaccess to redirect from the home directory to a particular directory and to index.php file, your plugin didn't updated my database until I removed my redirect on .htaccess, then it worked good and then I had to put back my .htaccess configuration.. weird. - so that's I think what may happened with this update.
